docsk8s/troubleshooting/ls-k8s-troubleshooting-methods.asciidoc

```diff
@@ -3,33 +3,49 @@
 
 There are a number of methods that you can use to diagnose the state of your Logstash and Kubernetes system, both in the event of any problems, and as part of a day-to-day approach to ensuring that everything is running as expected.
 
-<<ls-k8s-checking-resources>>
-<<ls-k8s-viewing-logs>>
-<<ls-k8s-connecting-to-a-container>>
+* <<ls-k8s-checking-resources>>
+* <<ls-k8s-viewing-logs>>
+* <<ls-k8s-connecting-to-a-container>>
+* <<ls-k8s-diagnostics>>
 
 [float]
 [[ls-k8s-checking-resources]]
 === Checking resources
 
 You can use the standard Kubernetes `get` and `describe` commands to quickly gather details about any resources in your Logstash and Kubernetes environment.
 
-Examples...
+`kubectl get pod logstash-7477d46bb7-4lcnv`
 
+`kubectl describe pod logstash-7477d46bb7-4lcnv`
```

Comment:
Maybe add some sample output from `describe pod` and what a failing logstash pod might look like?